cmd/root.go

```diff
@@ -40,16 +42,11 @@ func init() {
 }
 
 func logInit() {
-	Logfile := "/tmp/uyuni_iss_log.json"
-	lf, err := os.OpenFile(Logfile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
-	if os.IsNotExist(err) {
-		f, err := os.Create(Logfile)
-		if err != nil {
-			log.Error().Msg("Unable to create logfile")
-		}
-		lf = f
-	}
-	multi := zerolog.MultiLevelWriter(lf, os.Stdout)
+	syslogger, err := syslog.New(syslog.LOG_INFO|syslog.LOG_DEBUG|syslog.LOG_WARNING|syslog.LOG_ERR, "inter-server-sync")
+
+	syslogwriter := zerolog.SyslogLevelWriter(syslogger)
+
+	multi := zerolog.MultiLevelWriter(syslogwriter, os.Stdout)
```

Comment:
when running this code, the stdout will get duplicated messages. That is expected? 
It also gets following one: 
`Broadcast message from systemd-journald@hub-server (Wed 2021-05-26 13:33:27 WEST):`

Reply:
Yeah, I kept logs to stdout because otherwise, it will be too cumbersome for the user to see what is happening.

>Broadcast message from systemd-journald@hub-server (Wed 2021-05-26 13:33:27 WEST):

About this message, I didn't notice but I will take a look.